che-editors.yaml

```diff
@@ -0,0 +1,192 @@
+version: 1.0.0
+editors:
+  - repository:
+      url: https://github.com/eclipse/che-theia
+      revision: master
+      endpoints:
+        - name: "theia"
```

Comment:
how do we define that it'll be `next` and not `latest` ?

Reply:
also I don't find the `latest` definition

Reply:
We could use the `id` system, similar to for plugins?

Also about the repository fields, do we need revision? Some repos don't have anything interesting in them, or don't have tags create to match the release.

Reply:
Ok for `id` and let's remove repository for now if we don't see how to use it for now then.